web/src/features/fba/components/ZoneSummaryPanel.tsx

```diff
@@ -34,40 +35,37 @@ interface Props {
   fireZoneAreas: FireZoneArea[]
 }
 
-const ZoneSummaryPanel = (props: Props) => {
+const ZoneSummaryPanel = React.forwardRef((props: Props, ref: React.ForwardedRef<HTMLDivElement>) => {
   const classes = useStyles()
+  ZoneSummaryPanel.displayName = 'ZoneSummaryPanel'
 
   if (isUndefined(props.selectedFireZone)) {
     return <div></div>
   } else {
     return (
-      <Grid
-        container
-        alignItems={'center'}
-        direction={'column'}
-        spacing={2}
-        className={`${props.className} ${classes.wrapper}`}
-      >
-        <Grid item>
-          <Typography className={classes.zoneName}>{props.selectedFireZone.mof_fire_zone_name}</Typography>
-          <Typography className={classes.centreName}>{props.selectedFireZone.mof_fire_centre_name}</Typography>
+      <Container ref={ref} className={`${props.className} ${classes.wrapper}`}>
+        <Grid container alignItems={'center'} direction={'column'}>
+          <div>
```

Comment:
We'll want to maintain the existing `GridItem` components here instead of `div`s, it'll give us a bit more responsive flexibility: https://mui.com/material-ui/react-grid/

Reply:
Ok, I changed those to `div`s after talking with Darren about a display issue, but I think I've found a better way and can change them back to `Grid` items now